### DISCUSSION TITLE
Don't have access to copilot in VS

### DISCUSSION THREAD

**Discussion by smithago** [community/community#54981] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Followed the instructions [here](https://docs.github.com/en/copilot/getting-started-with-github-copilot?tool=visualstudio) 

Get this error after copying the code and authorizing co-pilot

GitHub Copilot 1.82.1.0 (v1.82.1.0@10c323cc0)
GitHub Copilot Agent v1.82.0
[DEBUG] [agent] [2023-05-09T19:47:56.736Z] Agent service starting
[DEBUG] [agent] [2023-05-09T19:47:56.953Z] Telemetry initialized
Please open https://github.com/login/device and enter the device activation code <edited by Staff> that has been copied to the clipboard.
[INFO] [auth] [2023-05-09T19:48:23.773Z] Invalid copilot token: missing token: 403 
You don't have access to GitHub Copilot.

How do I get this working?

**Comment by Akash1134** [comment, marked as answer]
Hi @smithago ,

Can you verify that you are logged in with the correct (official MSFT) account?

If you look to the left hand sidebar in VS Code, towards the bottom there is an accounts icon. Please follow that and log out of the GitHub account and then login again, making sure you are authenticating with <Official Login ID> in your browser.

Let me how it goes. 

Thanks!

**Comment by smithago** [comment]
I am using Visual Studio and not VS code. Is there a way to clear the old token? Even after I relogged in, It did not prompt me to re-autorize and is probably using a cached token and continues to throw 403
